src/components/sign-up.js

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import logo from '../assets/logo.svg';
+import facebook from '../assets/facebook.svg';
+import gmail from '../assets/gmail.svg';
+export default function SignUp() {
+  return (
+    <div
```

Comment:
Please wrap the whole component with a container

https://tailwindcss.com/docs/container